### ISSUE TITLE
PlaceholderField

### ISSUE THREAD

**Issue by stephendwolff** [divio/django-cms#4342]
I have a project which i'm trying to update to use django 1.8 (and djangocms 3.1), and have come across this following issue (i realise django 1.8 isn't quite supported yet).

I have a model similar to the example in the docs here: 
 - http://django-cms.readthedocs.org/en/latest/how_to/placeholders.html

```
from django.db import models
from cms.models.fields import PlaceholderField

def my_placeholder_slotname(instance):
    return 'placeholder_name'

class MyModel(models.Model):
    # your fields
    my_placeholder = PlaceholderField(my_placeholder_slotname)
    # your methods
```

When running `python manage.py migrate appname`, i get a migration created inside the djangocms cms app in my site-packages:

```
Migrations for 'cms':
  0013_auto_20150807_1037.py:
    - Change managers on pageuser
```

Is this a (non-south) migration that is missing from the current PyPI release (i'm just using the latest)? Or is it one that will be part of any django1.8 support patch?

**Comment by yakky** [comment]
@stephendwolff this is fixed in support/3.1 and will be released in 3.1.3 (due in a few days)

**Comment by stephendwolff** [comment]
ok - great. thanks for letting me know. I'll try using support/3.1 for now. Cheers

**Comment by yakky** [comment]
Fixed in #4232
